test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -1005,23 +930,23 @@ def __copy_dir_with_core(self, host, src_path, dest_path, core_dir,
         When a core file is found, capture the stack trace from it
 
         :param host: name of the host where the source is located
-        :type String
+        :type str
         :param src_path: path of the source directory
-        :type String
+        :type str
         :param dest_path: path of the destination directory
-        :type String
+        :type str
         :param core_dir: path to the directory to store core files' trace
-        :type String
+        :type str
         :param except_list: list  of files/directories (basenames) to exclude
-        :type List
+        :type list
         :param only_core: Copy over only core files?
-        :type Boolean
+        :type bool
         """
         dir_list = None
         try:
             dir_list = self.du.listdir(host, src_path, fullpath=False,
                                        sudo=self.sudo)
-        except:
+        except Exception:
```

Comment:
Actually I was doing this because only saying 'except' is bad practice, and 'Exception' catches most of the common Python API exceptions. I wanted to be more specific, but I, surprisingly, couldn't find any documentation for what exceptions are raised by the standard python functions.